### DISCUSSION TITLE
How to compare two detected images that are taken from diffirent cameras?

### DISCUSSION THREAD

**Discussion by tgrmev** [ultralytics/yolov5#10595] [Q&A]
Hello, I am using yolov5 to detect objects in two different camera. I would like to know which detected objects are the same in both camera. After implementation of yolov5, I cropped the bounded objects and implemented SIFT feature detection algorithm but it only works when two cameras are located like stereo camera. I am not sure but for my problem the best way is to use CNN to compare the images. How can I use your CNN network after detect the images with yolov5? As you can see from the image, I would like to get the information id2, cam1 is the similar object with id1,cam3. How can I do that?
![crops](https://user-images.githubusercontent.com/44428766/209563598-73448ce8-a4ce-4abf-8020-cbffb236b026.png)

**Comment by knoppmyth** [comment]
Have you looked at DeepSORT?
